demo/examples/04-jobs.js

```diff
@@ -0,0 +1,36 @@
+import { getJobsStatus, queuePrint } from 'inkmap';
+import { tap } from 'rxjs/operators';
+import { downloadBlob, getJobStatus } from '../../src/main';
+
+const root = document.getElementById('example-04');
+const btn = /** @type {CustomButton} */ root.querySelector('custom-button');
+const bars = /** @type {CustomProgresses} */ root.querySelector(
+  'custom-progresses'
+);
+const spec = /** @type {PrintSpec} */ root.querySelector('print-spec');
+
+// subscribe to the long-running observable
+// beware of managing the unsubscription
+getJobsStatus()
+  .pipe(
+    tap((jobs) => {
+      bars.jobsStatus = jobs;
+    })
+  )
+  .subscribe();
+
+btn.addEventListener('click', async () => {
+  // create a job, get a promise that resolves with the job id
+  const jobId = await queuePrint(spec.value);
+
+  getJobStatus(jobId).subscribe((printStatus) => {
```

Comment:
Is the Observable complete or do we need to unsubscribe somehow ?

Reply:
`getJobStatus` (single job) does complete eventually, same as the other examples.